systemuser.sh

```diff
@@ -48,6 +48,7 @@ JPY_CONFIG=$JUPYTER_CONFIG_DIR/jupyter_notebook_config.py
 echo "c.FileCheckpoints.checkpoint_dir = '$SCRATCH_HOME/.ipynb_checkpoints'"         >> $JPY_CONFIG
 echo "c.NotebookNotary.db_file = '$JUPYTER_LOCAL_PATH/nbsignatures.db'"     >> $JPY_CONFIG
 echo "c.NotebookNotary.secret_file = '$JUPYTER_LOCAL_PATH/notebook_secret'" >> $JPY_CONFIG
+echo "c.NotebookApp.iopub_data_rate_limit=5000000" >> $JPY_CONFIG
```

Comment:
What is this for?

Reply:
So that the kernel can send bigger images to the interface.